frontend/app/src/types/history/meta.ts

```diff
@@ -2,7 +2,8 @@ import { z } from 'zod';
 
 export const EntryMeta = z.object({
   ignoredInAccounting: z.boolean().nullish(),
-  customized: z.boolean().nullish()
+  customized: z.boolean().nullish(),
+  hasDetails: z.boolean().nullish()
```

Comment:
I did it because this entry meta is shared between trades, asset movements, and ledger actions.
But I think it's better to move it, and create new type to hold it for tx only.